crates/cxx-qt-gen/src/generator/rust/constructor.rs

```diff
@@ -278,7 +278,13 @@ pub fn generate(
             .map(|mut parameter| -> Result<_> {
                 if let FnArg::Typed(pat_type) = &mut parameter {
                     *pat_type.ty = syn_type_cxx_bridge_to_qualified(&pat_type.ty, type_names)?;
+                } else {
+                    // CODECOV_EXCLUDE_START
+                    // Unreachable as route_arguments_parameters are created with names like arg0: Type above
+                    unreachable!("Args here should not be able to be Receiver")
+                    // CODECOV_EXCLUDE_STOP
                 }
+                // Should this be an error if FnArg::Receiver is passed?
```

Comment:
this comment can be removed? there is unreachable now above?

Reply:
yea the args it is getting are built in the block above and get given idents like arg0, arg1, so they could never be self: T or whatever

Reply:
cool